eftest/src/eftest/report/pretty.clj

```diff
@@ -30,12 +31,19 @@
   "The divider to use between test failure and error reports."
   "\n")
 
-(defn- testing-vars-str [{:keys [file line]}]
-  (let [test-var (first test/*testing-vars*)]
-    (str (:clojure-frame *fonts*) (-> test-var meta :ns ns-name) "/"
-         (:function-name *fonts*) (-> test-var meta :name) (:reset *fonts*)
-         (when (or file line)
-           (str " (" (:source *fonts*) file ":" line (:reset *fonts*) ")")))))
+(defn- testing-scope-str [{:keys [file line]}]
+  (let [[ns scope] report/*testing-path*]
+    (str
+      (cond
```

Comment:
Can you tweak the indentation here so it's one space back? I'm following the Clojure style guide.